server/src/service-layer/controllers/UserController.ts

```diff
@@ -75,4 +79,62 @@ export class UsersController extends Controller {
     await Promise.all(editPromises)
     this.setStatus(200)
   }
+
+  // ticket 202 - endpoint to demote/promote users
+  @SuccessResponse("200", "Promoted user")
+  @Security("jwt", ["admin"])
+  @Put("promote")
+  // set user membership to "member"
+  public async promoteUser(
+    @Body() requestBody: promoteUserRequestBody
+  ): Promise<void> {
+    const userService = new UserDataService() // create a new data service
+    const authService = new AuthService()
+    // attempt to fetch user
+    const user: UserAdditionalInfo = await userService.getUserData(
+      requestBody.uid
+    )
+    if (!user) return this.setStatus(400) // bad request
+    if (user.membership === "admin") return this.setStatus(403) // admin forbidden
```

Comment:
we should not be checking this in firestore database, we need to check the `customClaims option` on the user. Should add a method in `AuthService` thats uses `auth.getUser(uid)`, the `userRecord` returned will have a `customClaims` property